### DISCUSSION TITLE
V1.1.6 - Windows - Forlders are beging created with a <space> before the name

### DISCUSSION THREAD

**Discussion by Freisgd** [enzo1982/freac#302] [Help & Support]
Not sure why this is happening, but many of the folders that are created during the ripping process contain an erroneous <space> in the name.  
It's a 3 step process to remove it as you have to make a new folder w/o the space as windows is stupid and says that [ Folder] is the same name as [Folder], and yet alphabetically it puts the folders with spaces first, you can't just delete the <space>.
Has anyone else run into this?
I created a Power Shell program to remove to beginning space from file names, but it doesn't work on folder names. :(
It doesn't put the <space> on all folders, so I don't know if its getting bad data from the CDDB or why the <space> exists.

**Comment by enzo1982** [comment]
Please check the output filename pattern in the config dialog. From your description, I would guess that you have a space at the start of the output filename pattern or directly after a folder separator (`\` or `/`).

**Reply by Freisgd** [reply]
Wow, Thank you.  I feel so Stupid "<albumartist> \ <album> \ <track> - <artist> - <title>" was what was in the filename.  Changed it to "<albumartist>\<album>\<track> - <artist> - <title>".  Much better! Weird thing is that I do not ever remember setting this, just choosing from an list of options.  Again sorry for any inconvenience.
